apps/web/src/components/WorkforceAdjustment/Warning.tsx

```diff
@@ -13,13 +13,13 @@ interface WarningProps {
 }
 
 const Warning = ({ children, className, isError }: WarningProps) => (
-  <Well
+  <Notice
     color={isError ? "error" : "warning"}
     className={warning({ class: className })}
   >
     <ExclamationTriangleIcon className="mt-1 size-4 shrink-0" />
     <div className="flex-grow">{children}</div>
-  </Well>
+  </Notice.Root>
 );
```

Comment:
This could just be:

```tsx
<Notice.Root color={isError ? "error" : "warning">
  <Notice.Title icon={ExclamationTriangleIcon} className="font-normal">{children}</Notice>
</Notice.Root>
```